web-registry/src/components/organisms/ProjectTopSection.tsx

```diff
@@ -222,23 +218,33 @@ function ProjectTopSection({
   const videoURL = metadata?.['http://regen.network/videoURL']?.['@value'];
   const landStewardPhoto =
     metadata?.['http://regen.network/landStewardPhoto']?.['@value'];
+  const area =
+    metadata?.['http://regen.network/size']?.[
+      'http://qudt.org/1.1/schema/qudt#numericValue'
+    ]?.['@value'] ||
+    metadata?.['http://regen.network/projectSize']?.[
+      'http://qudt.org/schema/qudt/numericValue'
+    ]?.['@value'];
   const unit: qudtUnit | undefined =
     metadata?.['http://regen.network/size']?.[
       'http://qudt.org/1.1/schema/qudt#unit'
+    ]?.['@value'] ||
+    metadata?.['http://regen.network/projectSize']?.[
+      'http://qudt.org/schema/qudt/#unit'
```

Comment:
eventually we'll need to update the old projects metadata to use the most up-to-date qudt reference (without the version 1.1) but that can be addressed at a later point
```suggestion
    ]?.['@value'] ||
    metadata?.['http://regen.network/projectSize']?.[
      'http://qudt.org/schema/qudt/unit'
```